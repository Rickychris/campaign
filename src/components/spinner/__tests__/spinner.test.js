import React from "react";
import ReactDom from "react-dom";
import { render, cleanup } from "@testing-library/react";
import Spinner from "../spinner";

describe("Spinner tests", () => {
  beforeEach(cleanup);

  it("renders the Spinner", () => {
    const div = document.createElement("div");
    ReactDom.render(<Spinner></Spinner>, div);
  });

  it("matches snapshot", () => {
    const { getByTestId } = render(<Spinner />);
    expect(getByTestId("spinner")).toMatchSnapshot();
  });
});
