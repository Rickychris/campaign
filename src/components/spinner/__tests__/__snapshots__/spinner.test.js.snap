// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner tests matches snapshot 1`] = `
<div
  class="root"
  data-testid="spinner"
>
  <div
    class="spinner"
  />
</div>
`;
